addons/rating_project/views/project_view.xml

```diff
@@ -32,7 +36,7 @@
             <field name="inherit_id" ref="project.view_task_form2"/>
             <field name="arch" type="xml">
                 <xpath expr="//button[@name='toggle_active']" position="before">
-                    <button name="%(rating_rating_action_task)d" type="action"
+                    <button name="action_open_rating_view" type="object"
```

Comment:
no.. we can't use same exisitng action defined in xml. because we already set primary mode view based on product view action..(which has same model but res_id and res_model_id area different.)